test/known_issues/test-inspector-cluster-port-clash.js

```diff
@@ -1,6 +1,7 @@
 // Flags: --inspect=0
 'use strict';
 const common = require('../common');
+common.crashOnUnhandleRejection();
```

Comment:
can you please move this to line 26 below all the require statements

Reply:
fix that.